kolibri/plugins/coach/assets/src/csv/fields.js

```diff
@@ -216,10 +228,10 @@ export function tally() {
   ];
 }
 
-export function timeSpent(key, label = 'timeSpentLabel') {
+export function timeSpent(key, label = coreStrings.$tr('timeSpentLabel')) {
```

Comment:
I'd prefer to still invoke the $tr method in function scope rather than module scope (which it will for a default) - just in case there's a race condition here between i18n setup and execution of this module code.

It's possible that I am remembering a gotcha that I have since fixed, but out of abundance of caution let's keep the invocation in function scope for now.